packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -534,6 +530,32 @@ const tenantsRouter = (
           return res.status(errorRes.status).json(errorRes).end();
         }
       }
+    )
+    .delete(
+      "/tenants/:tenantId/attributes/verified/:attributeId",
+      authorizationMiddleware([ADMIN_ROLE]),
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
+        try {
+          const tenant = await tenantService.revokeVerifiedAttribute(
+            {
+              tenantId: unsafeBrandId(req.params.tenantId),
+              attributeId: unsafeBrandId(req.params.attributeId),
+              revokerId: req.ctx.authData.organizationId,
+              correlationId: req.ctx.correlationId,
+            },
+            ctx.logger
```

Comment:
```suggestion
            },
            ctx
```
You can get everything from `ctx`